### PR TITLE
[wpmlbridge-301] Unsync documents on non indexable stati.

### PR DESCRIPTION
**Pull request by decodekult**
- Unsync documents with a status not supported by their indexes.

https://github.com/OnTheGoSystems/wpml-elasticpress/issues/49

### CODE REVIEW

File: src/Sync/Singular.php [hunk 1]
```diff
@@ -234,4 +236,29 @@ public function clearIndexLanguageByMeta( $metaId, $postId ) {
...
+		if ( $oldStatus === $newStatus ) {
+			return;
+		}
+
+		$postIndexable = $this->indexables->get( Constants::INDEXABLE_SLUG_POST );
+		if ( false === $postIndexable ) {
+			return;
+		}
+
+		$indexablePostStati = $postIndexable->get_indexable_post_status();
```
Comment (Pawel-H-OTGS) [inline src/Sync/Singular.php:254]:
```suggestion
		$indexablePostStatus = $postIndexable->get_indexable_post_status();
```

Comment (Pawel-H-OTGS) [reply src/Sync/Singular.php:254]: there is also `Stati` in the commit so I don't know if it's a typo or intentional :)

everything else is great :+1:

**Comment by jamalbou** [comment]
+1 @decodekult

**Review by jamalbou — APPROVED** [review]
+1 @decodekult

File: src/Sync/Singular.php (hunk 1 above)
Comment (dgwatkins) [reply src/Sync/Singular.php:254]: I understand stati is the plural form of status (as opposed to statuses which doesn't really exist). I saw it here:

https://developer.wordpress.org/reference/functions/get_post_stati/

**Pull request merged**